docs/source/user_guide/operators.rst

```diff
@@ -307,12 +307,9 @@ DPF provides three main types of operators:
 Operators for importing or reading data
 ***************************************
 
-These operators provide for reading data from solver files or from standard file types:
-
-- For MAPDL, supported solver file formats include RST, MODE, RFRQ, and RDSP.
-- For LS DYNA, D3PLOT files are supported.
-- For Fluent and CFX, CAS.H5, DAT.H5, RES, and FLPRJ files are supported.
-- For Abaqus, ODB files are supported.
+These operators provide for reading data from solver files or from standard file types
+such as .RST (MAPDL), .D3Plot (LS DYNA), .CAS.H5/.DAT.H5 (Fluent), .CAS.CFF/.DAT.CFF (CFX)
+or .OBD (Abaqus).
```

Comment:
@MichaelNale .odb to confirm, I heard we actually do not support it

Reply:
Yes I made the changes in the table but not here.